src/animation.rs

```diff
@@ -31,7 +31,7 @@
 //!
 //! First, we create a [`Mixer`] to play our animation.
 //!
-//! ```rust,no_run
+//! ```rust,no_run,ignore
```

Comment:
why is `ignore` needed here? Perhaps, we need a TODO comment to rewrite those docs without the use of gtf-loader

Reply:
The TODOs would be difficult to notice.

```rust
//! ```rust,no_run,ignore
//! # // TODO: Rewrite test
//! ...
//! ```

Reply:
Oh, perhaps just at the crate root instead?

Reply:
yep, that would do